### PR TITLE
Add worker api

### PR DESCRIPTION
**Pull request by pajowu**
- ✨ Add api to create workers
- ✨ Add endpoint to list workers

### CODE REVIEW

File: frontend/src/openapi-schema.ts [hunk 1]
```diff
@@ -445,6 +458,33 @@ export interface components {
...
+       * Format: date-time
+       */
+      last_seen?: string;
+      /** Name */
+      name: string;
+      /** Token */
+      token: string;
+    };
+    /** WorkerBase */
+    WorkerBase: {
```
Comment (phlmn) [inline frontend/src/openapi-schema.ts:479]: `id` should probably be included here

Comment (pajowu) [reply frontend/src/openapi-schema.ts:479]: No, only a `transcribee_backend.models.worker.Worker` has a id. The worker base doesn't

Comment (phlmn) [reply frontend/src/openapi-schema.ts:479]: In know, but would be good to have in the response

Comment (pajowu) [reply frontend/src/openapi-schema.ts:479]: Why? The worker id is only used internally in the backend. The only thing needed via the api is the token / name

**Comment by phlmn** [comment]
Can I have worker deregistration/deletion? 🥺👉👈

**Review by phlmn — APPROVED** [review]

**Pull request merged**